android/titanium/src/java/org/appcelerator/titanium/io/TiFile.java

```diff
@@ -95,6 +97,71 @@ public boolean isWriteable()
 		return file.canWrite();
 	}
 
+	@Override
```

Comment:
Why not just make use of the `write` methods that already exist with an `append` flag?

```Java
@Override
public boolean append(Object data) throws IOException
{
	try {
		if (data instanceof String) {
			write((String) data, true);
		} else if (data instanceof TiFileProxy) {
			// write((TiFileProxy) data, true); // implement an append flag into this
		} else if (data instanceof TiBlob) {
			write((TiBlob) data, true);
		}
		return true;
	} catch (IOException e) {
		Log.e(TAG, "Error appending: ", e);
	}
	return false;
}
```